### PR TITLE
zh-CN: lite/microcontrollers/get_started.md

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

File: site/zh-cn/lite/microcontrollers/get_started.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
+# 微控制器入门
+
+本文档将帮助您开始使用用于微控制器的Tensorflow Lite。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/microcontrollers/get_started.md]: 中英文请使用空格分隔

File: site/zh-cn/lite/microcontrollers/get_started.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+-   [由 Tensorflow 提供技术支持的 SparkFun Edge(Apollo3 Blue)](https://www.sparkfun.com/products/15170)
+-   [STM32F746G 探索板（Discovery Board）](https://www.st.com/en/evaluation-tools/32f746gdiscovery.html)
+-   Mac OS X
+
+注意：若要开始使用 SparkFun Edge 板，我们建议遵循[“在使用 SparkFun Tensorflow 的微控制器上进行机器学习”](https://codelabs.developers.google.com/codelabs/sparkfun-tensorflow)中所描述的流程,这是一个向您介绍开发工作流程的代码实验室（codelab）。
+
+### 微视觉示例
+
+本示例展示了如何使用 Tensorflow Lite 运行一个250千字节的神经网络来识别由摄像机拍摄的图像中的人。该示例被设计成可以在具有少量内存的系统上运行，如微控制器和 DSP。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/microcontrollers/get_started.md]:
```suggestion
本示例展示了如何使用 Tensorflow Lite 运行一个 25 万字节的神经网络来识别由摄像机拍摄的图像中的人。该示例被设计成可以在具有少量内存的系统上运行，如微控制器和 DSP。
```

File: site/zh-cn/lite/microcontrollers/get_started.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+```
+
+在示例其他部分中，使用了一个更加复杂的算法来平滑多帧的识别结果。该部分在 [recognize_commands.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/examples/micro_speech/recognize_commands.h) 中有所定义。在处理任何连续的数据流时，也可以使用相同的技术来提高可靠性。
+
+## 下一步
+
+创建并运行示例后，请阅读以下文档：
+
+*   在[“构建与转换模型”](build_convert.md)中了解如何使用模型。
+*   在[“了解C++库”](library.md)中了解更多关于C++库的内容。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/microcontrollers/get_started.md]:
```suggestion
*   在[“了解C++库”](library.md)中了解更多关于 C++ 库的内容。
```

Comment (Berrywrq) [reply site/zh-cn/lite/microcontrollers/get_started.md]: OK, I have made changes to the above mentioned problems, please checkout

**Pull request merged**